tools/issue-labeler/src/CreateMikLabelModel/CreateMikLabelModel.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\eng\Versions.props" />
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Hubbup.MikLabelModel\Hubbup.MikLabelModel.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Octokit" Version="13.0.1" />
```

Comment:
That's interesting.    Are we actually using versions from `Versions.props` or overriding all over?

Reply:
I used it to centralize some versions as of new push, but before that it was just being overridden.